daphne/src/roles.rs

```diff
@@ -227,6 +252,11 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
             return Err(DapAbort::UnauthorizedRequest);
         }
 
+        // Check whether the DAP version in the request is not Unknown.
```

Comment:
This check should be the very first thing (even before authorization).

Reply:
Moved this before authorization.